src/components/notifications/content/ChallengeReward.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import { useTheme } from '../../../utils/theme'
+import {
+  ChallengeReward as ChallengeRewardType,
+  ChallengeRewardID
+} from 'store/notifications/types'
+import TwitterShare from './TwitterShare'
+
+const styles = StyleSheet.create({
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column'
+  },
+  text: {
+    fontFamily: 'AvenirNextLTPro-Bold',
+    fontSize: 16,
+    marginBottom: 8
+  }
+})
+
+type ChallengeRewardProps = {
+  notification: ChallengeRewardType
+  onGoToRoute: (route: string) => void
+}
+
+const challengeInfoMap: Record<
+  ChallengeRewardID,
+  { title: string; amount: number }
+> = {
+  'profile-completion': {
```

Comment:
i wonder if we can/should pull these in via optimizely. e.g. if we change the 7 days to 6 days or something

Reply:
💯 I agree. This same type of config is in 3 places now so I want to put it somewhere central for them all to read from instead

Reply:
maybe the client could fetch from discovery

Reply:
the more i think about it, the more i like this. wdyt?

Reply:
also down to merge this as is first

Reply:
Cool will merge now and refactor in another PR. And yea I'd like to discuss that approach 👍